### ISSUE TITLE
Chart Interaction

### ISSUE THREAD

**Issue by swapnalithakarpspc** [wet-boew/wet-boew#8942]
Hi Is the following type of chart interaction possible with Flot?

![image](https://user-images.githubusercontent.com/71705355/94030965-62058880-fd8c-11ea-86ad-90a77329db84.png)
![image](https://user-images.githubusercontent.com/71705355/94031050-78abdf80-fd8c-11ea-8ed1-9572e7b87445.png)

Please let me know.
Thanks.

**Comment by duboisp** [comment]
Yes something similar is possible, but will require you to add some flot plugin and test for accessibility.

I don't which plugin that can be used, but you can check the available flot plugin here: http://www.flotcharts.org/flot/examples/ 
You will need to add some custom javascript in your page to enable those plugin. See here: https://github.com/wet-boew/wet-boew/issues/4466#issuecomment-75772427
Perform a full WCAG assessment of your page after the new flot plugin are added.
